### PR TITLE
feat(user-notification + cms-client): Summer cleaning

### PR DESCRIPTION
**Pull request by rafnarnason**
property name cleaning
phasing out body and datacopy for externalbody and internalbody in code
removing unused properties
improved logging
cleaning up linter warnings in client-cms
updating user-notification gql queries

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

## Summary by CodeRabbit

- **New Features**
  - Enhanced notification properties for better clarity and usability, including new property names and structures.

- **Bug Fixes**
  - Removed redundant properties from notification objects to streamline data handling.

- **Documentation**
  - Updated environment variable configuration instructions and manual schema update command in the CMS client README.

- **Refactor**
  - Renamed several properties across various modules for consistency and improved code readability.

- **Chores**
  - Added specific rule overrides in `.eslintrc.json` for better linting control on specific files.
  - Introduced a new `cleanString` utility function for string manipulation.

### CODE REVIEW

**Comment by rafnarnason** [comment]
coderabbit ai pause

**Comment by rafnarnason** [comment]
coderabbitai pause

**Review by kksteini — APPROVED** [review]

**Review by thordurhhh — APPROVED** [review]
🙌

File: apps/services/user-notification/src/app/modules/notifications/utils.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import type { Locale } from '@island.is/shared/types'
 export const isDefined = <T>(x: T | null | undefined): x is T => x != null
 export const mapToLocale = (locale: string): Locale =>
   locale === 'en' ? 'en' : 'is'
-export const mapToContentfulLocale = (locale: Locale): string => {
-  return locale === 'en' ? 'en' : 'is-IS'
-}
+export const mapToContentfulLocale = (locale: Locale): string =>
+  locale === 'en' ? 'en' : 'is-IS'
+export const cleanString = (str: string): string =>
+  str.replace(/\s+/g, ' ').trim()
```
Comment (baering) [inline apps/services/user-notification/src/app/modules/notifications/utils.ts:8]: ~This exact function already exists in api as a (non exported) [helper](https://github.com/island-is/island.is/blob/main/libs/api/domains/notifications/src/utils/helpers.ts#L8) as well, maybe it's generic enough to be moved into shared/utils instead 🤷

**Review by baering — APPROVED** [review]

File: apps/services/user-notification/src/app/modules/notifications/utils.ts (hunk 1 above)
Comment (rafnarnason) [reply apps/services/user-notification/src/app/modules/notifications/utils.ts:8]: will discuss with api team and implement in the next hnipp planning

**Pull request merged**